py/VERSION.txt

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
I don't think these files are used anywhere - `tfm-regression-non-secure.txt` and `tfm-regression-secure.txt`

Reply:
The spec file too can be deleted then.